de-identification-entry-spark/src/main/java/com/ibm/whc/deid/external/spark/udf/DeIdSynapseUDF.java

```diff
@@ -0,0 +1,110 @@
+/*
```

Comment:
Do we need to name this UDF DeId`Synapse`UDF?  It doesn't look like there is any Synapse specific code in here.  I wonder if getting the mount method getSynapseMountPoint  would work in other spark environments.

Reply:
The location of mounted storage accounts in a directory related to the current job number seems to be Synapse-specific.  I doubt this would work in other environments.